src/include/units/bits/one_rep.h

```diff
@@ -0,0 +1,60 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2018 Mateusz Pusz
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#pragma once
+
+#include <units/bits/basic_concepts.h>
+#include <cstdint>
+#include <type_traits>
+
+namespace units::detail {
+
+struct one_rep {
+  template<QuantityValue Rep>
+  [[nodiscard]] friend constexpr Rep operator*(const Rep& lhs, one_rep)
+  {
+    return lhs;
+  }
+  template<QuantityValue Rep>
+  [[nodiscard]] friend constexpr Rep operator/(const Rep& lhs, one_rep)
+  {
+    return lhs;
+  }
+
+  constexpr operator std::intmax_t() const noexcept { return 1; }
+
+  bool operator==(const one_rep&) const = default;
+  auto operator<=>(const one_rep&) const = default;
+};
```

Comment:
I must say I really like the `one_rep` approach. I did not think about this by myself. Good job!